### DISCUSSION TITLE
can i have a clean build?

### DISCUSSION THREAD

**Discussion by Alone0316** [ROM-builders/temporary#12986] [General]
Can i have a clean build? as i am facing some problem with compiling on cirrus, but locally it compiles fine without any problem

**Comment by Apon77** [comment]
Link?

**Reply by Alone0316** [reply]
https://cirrus-ci.com/task/6333480266104832

**Reply by Apon77** [reply]
I dont think clean can solve the issue. please find the solution in another way. because after your build, there are many successful builds.

**Reply by Alone0316** [reply]
I asked 2 people to compile Los with same trees and it compiled fine for them on their server that's why i asked for clean build, i was facing this problem with 1 person where it was dirty build, i asked him for clean build and it worked!
@Apon77
